libafl/src/mutators/tuneable.rs

```diff
@@ -135,7 +132,7 @@ where
                 iters
             } else {
                 // fall back to random
-                1 << (1 + state.rand_mut().below(self.max_stack_pow))
+                1 << (1 + state.rand_mut().zero_upto(self.max_stack_pow))
```

Comment:
Why? A 0 here makes no sense

Reply:
why 0 makes no sense?
you can tell it to do up to 2 iterations = (1 << 1) at maximum

Reply:
if you make it nonzero then you can't do this because the minimum upperbound is 1 << (1 + 1) = 4

Reply:
Ah yeah math is hard.